### ISSUE TITLE
Rework source tree

### ISSUE THREAD

**Issue by paceholder** [paceholder/nodeeditor#18]
The current way of implementing the headers in the `include` folder makes the use of installed library impossible. This must be reworked to the structure as follows:
```
include/
    nodes/
        public *.hpp files
src/
    private *.hpps
    *.cpps
```

**Comment by Quincunx271** [comment]
Was this issue addressed by PR #142 ?

**Comment by paceholder** [comment]
@Quincunx271, right, closing it.
